installation/single_machine_docker.rst

```diff
@@ -54,7 +54,7 @@ The first time you start the cluster it builds its containers. Subsequent startu
 
 **3. Verify that installation has succeeded**
 
-To verify that the installation has succeeded we check that the master node has picked up the desired worker configuration. First start the psql shell on the master node:
+To verify that the installation has succeeded we check that the coordinator node has picked up the desired worker configuration. First start the psql shell on the coordinator (master) node:
 
 .. code-block:: bash
 
```

Comment:
It's hard-coded in our docker compose file https://github.com/citusdata/docker/blob/master/docker-compose.yml#L5